### ISSUE TITLE
[icerec] Incremental Release V05-01-08 (Trac #2084)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2076]
New L3 scripts for the Muon and Cascade WG are available for pass2 and season 2017.

I need a new incremental icerec release with new releases of those trunks:

* http://code.icecube.wisc.edu/svn/projects/level3-filter-cascade/trunk/
* http://code.icecube.wisc.edu/svn/projects/level3-filter-muon/trunk/

Migrated from https://code.icecube.wisc.edu/ticket/2084
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:38", 
    "description": "New L3 scripts for the Muon and Cascade WG are available for pass2 and season 2017.\n\nI need a new incremental icerec release with new releases of those trunks:\n\n* http://code.icecube.wisc.edu/svn/projects/level3-filter-cascade/trunk/\n* http://code.icecube.wisc.edu/svn/projects/level3-filter-muon/trunk/", 
    "reporter": "joertlin", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067278746682", 
    "component": "combo reconstruction", 
    "summary": "[icerec] Incremental Release V05-01-08", 
    "priority": "major", 
    "keywords": "", 
    "time": "2017-08-28T18:53:40", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170905T12:40:35: 
* kjmeagher changed status from "new" to "closed"
* kjmeagher commented:

> In changeset 157948:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="157948"
> incremetal release of icecrec:
> * level3-filter-lowen: fix import statements
> * level3-filter-cascade: Replace SeededRT with STTools
> 
> closes #2076
> ```


* kjmeagher changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:38: 
* olivas commented:

> Milestone September 2017 deleted


* olivas changed milestone from "September 2017" to ""
